### DISCUSSION TITLE
bounce+number+@mydomain.com problem.

### DISCUSSION THREAD

**Discussion by mrghxst** [simple-login/app#705] [Self-hosting question] [answered]
Sorry for spamming so many questions. It's just very hard for people like me who barely understand English to find any old question on the same topic (I tried, sorry).
My problem is that when I send/receive an email, it is always sent from a random address like bounce+anynumber+@myactualdomain.com.
Example: bounce+32+@myshield.com
That's not a problem, (I've seen your instance also uses what looks like random strings to send received emails). 
But when using reverse aliases this also happens (in my self-hosted instance), which I didn't think was the point of the reverse aliases.
Thanks!

**Comment by nguyenkims** [comment]
Hi the `bounce+...` address is actually used as the envelope sender and not the email From: header. Therefore it isn't visible in an email client.

This technique is used for handling bounces later and is called VERP https://en.wikipedia.org/wiki/Variable_envelope_return_path

**Comment by mrghxst** [comment, marked as answer]
I think this isnt working for me. 
Could be because I am using some random smtp relay which makes this fail ??
![download](https://user-images.githubusercontent.com/67842132/143618288-1bac616f-fab7-4af1-9d86-007f4cf1c1b9.jpeg)
![Screenshot (38)](https://user-images.githubusercontent.com/67842132/143618893-c5341e9d-1f53-4db8-b7b8-379dec04e0ad.png)
![Screenshot (39)](https://user-images.githubusercontent.com/67842132/143618896-6f51c801-e5d2-4fc6-9c92-617d55a8b96e.png)
it is called mailazy. Could change and try any other relay like the amazon if you think this is the problem or just re-install server and check if it works then ?

**Reply by nguyenkims** [reply]
It's possible that a SMTP relay doesn't respect the difference between envelope sender and header sender. We've never tried with SES though as we send emails directly from our server ...

**Reply by mrghxst** [reply]
Thank you for all your help!  My SimpleLogin testing server seems to work perfectly. I changed smtp relay and now it works perfect.
